site.yml

```diff
@@ -104,9 +142,15 @@
         - clang
         - gcc
         - mono-devel
+        state: latest
+      become: true
+    - name: Install python-pip
```

Comment:
It might make sense to remove the Python 2 analysis requirements for all operating systems in a future release as less distributions will be including them and users will hopefully be less commonly using Python 2 for their own source code.